### ISSUE TITLE
What do we need to do to preserve Teleporter support?

### ISSUE THREAD

**Issue by bryophyta** [guardian/dotcom-rendering#7307]
Things that don't work:
- [ ] The grid long press to access image is broken
  <img width="825" alt="Screenshot 2023-03-28 at 15 20 41" src="https://user-images.githubusercontent.com/31692/228268923-e42d926a-1a50-49ad-ae49-4e347a99f57b.png">

---

As a note, @guardian/digital-cms is the owner of this tool if we need help.

**Comment by paperboyo** [comment]
> grid right click

That’s a long-press (left click).

- [ ] hover over video atoms is broken too:
    ![image](https://user-images.githubusercontent.com/6032869/228271370-09135d02-23c1-49c3-87a9-180eb217c886.png)

**Comment by paperboyo** [comment]
Just a note that this is getting more and more important as the DCR front rollout progresses…

**Comment by rhiannareechaye** [comment]
@twrichards has kindly volunteered to take a look at this for us

**Comment by twrichards** [comment]
> @twrichards has kindly volunteered to take a look at this for us

I've kindly volunteered to pair with someone on the DCR team.

**Comment by twrichards** [comment]
... suspect we'll need to address https://github.com/guardian/gustaf/issues/88 before we can release any changes to gustaf (the browser extension portion of teleporter) which I suspect these fixes tracked in this issue will be primarily if not exclusively in the UI/extension

**Comment by alinaboghiu** [comment]
TODO 


- [x] make image long press work for both frontend and DCR fronts
- [ ] display links for all image in slideshows 
- [ ] videos 
- [ ] CODE environment 
- [ ] CODE viewer
- [ ] manifest version
- [ ] node version 
- [ ] lightbox

**Comment by rhiannareechaye** [comment]
Hey @alinaboghiu this was closed but from your list it looks like there's still stuff do to here so I've reopened?

**Comment by alinaboghiu** [comment]
Yep it closes automatically when PRs get merged but we'll have multiple PRs so let's leave it open :D

**Comment by rhiannareechaye** [comment]
Hey @alinaboghiu we're going to remove this from our board as we aren't actively working on it
